pkg/pillar/cmd/volumemgr/handleverifier.go

```diff
@@ -92,51 +124,9 @@ func MaybeRemoveVerifyImageConfig(ctx *volumemgrContext, objType, imageSha strin
 	log.Infof("MaybeRemoveVerifyImageConfig done for %s", imageSha)
 }
 
-func publishVerifyImageConfig(ctx *volumemgrContext, objType string,
-	config *types.VerifyImageConfig) {
-
-	key := config.Key()
-	log.Debugf("publishVerifyImageConfig(%s/%s)", key, objType)
-	pub := ctx.publication(*config, objType)
-	pub.Publish(key, *config)
-}
-
-func unpublishVerifyImageConfig(ctx *volumemgrContext, objType string, key string) {
-
-	log.Debugf("unpublishVerifyImageConfig(%s)", key)
-	pub := ctx.publication(types.VerifyImageConfig{}, objType)
-	c, _ := pub.Get(key)
-	if c == nil {
-		log.Errorf("unpublishVerifyImageConfig(%s) not found for %s", key, objType)
-		return
-	}
-	pub.Unpublish(key)
-}
-
-func publishPersistImageConfig(ctx *volumemgrContext, objType string,
-	config *types.PersistImageConfig) {
-
-	key := config.Key()
-	log.Debugf("publishPersistImageConfig(%s)", key)
-	pub := ctx.publication(*config, objType)
-	pub.Publish(key, *config)
-}
-
-func unpublishPersistImageConfig(ctx *volumemgrContext, objType string, key string) {
-
-	log.Debugf("unpublishPersistImageConfig(%s)", key)
-	pub := ctx.publication(types.PersistImageConfig{}, objType)
-	c, _ := pub.Get(key)
-	if c == nil {
-		log.Errorf("unpublishPersistImageConfig(%s) not found for %s", key, objType)
-		return
-	}
-	pub.Unpublish(key)
-}
-
 func handleVerifyImageStatusModify(ctxArg interface{}, key string,
```

Comment:
Why change the pattern and make this harder to understand? All of the updates and publishes should be done from updateVolumeStatus this handleX should not need to change.
updateVolumeStatus does a lookup and if it needs to update FileLocation or Size it should be done there.

WAIT, you are updating VerifyImageConfig. Why do you ever need to do that?
I'm trying to understand your pattern.

Reply:
When we create a VerifyImageConfig initially, the FileLocation will be pointing to the file at pending dir.
After verifier moves the file to verified dir, it sends the updated path to volumeMgr. Then volumeMgr will update VerifyImageConfig with the new path where the FileLocation will be pointing to the file at verified dir.

This was done to keep VerifyImageConfig and VerifyImageStatus in sync.

Reply:
> This was done to keep VerifyImageConfig and VerifyImageStatus in sync.

There is no requirement to do that at all.
Status is updated without updating config across the board.

Reply:
Removed config update part.